backend/api/appsettings.Development.json

```diff
@@ -41,6 +41,6 @@
     "AccountName": "flotilladevsa"
   },
   "Database": {
-    "UseInMemoryDatabase": true
+    "UseInMemoryDatabase": false

```

Comment:
For my curiosity, why is this needed/ what is the benefit of this?  "UseInMemoryDatabase": false

Reply:
I also have the same question, wouldn't it make sense to keep it as true given that these are the settings that are loaded when we run flotilla locally?
If we set it to false we will be polluting the dev database when doing local testing, correct?

Reply:
I think this should probably still stay as true